WcaOnRails/app/models/wcif.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Wcif < ApplicationRecord
+  belongs_to :competition
+
+  # See https://github.com/thewca/worldcubeassociation.org/wiki/wcif
+  def json
```

Comment:
Cool, I'm glad you noticed this. `json` and `json_hash` are the best I could come up with, but I don't particularly like them. Here are the things we need names for:

1. The database column in the `wcifs` table that will hold the JSON WCIF data for a competition.
2. A method to get the WCIF object for a competition (currently `def json`). This will read the string column from 1), convert it to json, and merge it with whatever structured data for the competition we already have in our database.
3. A method to set the WCIF for a competition (currently `def json=(new_json)`)
4. A method to get the hash/checksum of the JSONified version of 2).

(To be fair, every ActiveRecord class has methods with the same name as db columns. Not going to claim that what I'm doing is a standard or readable thing, though.)